packages/quantic/force-app/main/default/lwc/quanticCategoryFacet/quanticCategoryFacet.html

```diff
@@ -1,34 +1,52 @@
 <template>
     <template if:true={hasParentsOrValues}>
         <lightning-card title={label}>
-            <ul class="slds-has-dividers_around-space slds-m-bottom_medium">
-                <template if:true={hasParents}>
-                    <lightning-button variant="base" label={labels.clear} title={labels.clear} icon-name="utility:clear"
-                                      class="slds-m-left_medium" onclick={reset}></lightning-button>
-                </template>
-                <template for:each={parents} for:item="v">
-                    <li class="slds-m-horizontal--medium slds-m-vertical--x-small slds-grid" key={v.value}>
-                        <span class="nowrap slds-text-title_bold">{v.value}</span>
+            <lightning-button-icon
+                class="slds-var-p-bottom_small facet__collapse"
+                slot="actions"
+                onclick={toggleFacetVisibility}
+                icon-name={collapseIconName}
+                alternative-text="Collapse/show facet"
+                variant="bare">
+            </lightning-button-icon>
+            <template if:false={isCollapsed}>
+                <ul class="slds-has-dividers_around-space">
+                    <template if:true={hasParents}>
+                       <li class="slds-var-m-horizontal_medium slds-grid">
+                           <div onclick={reset}>
+                                <lightning-button-icon variant="bare" icon-name="utility:chevronleft"></lightning-button-icon>
+                                <span class="nowrap">{labels.allCategories}</span>
+                           </div> 
+                       </li> 
+                    </template>
+                    <template for:each={nonActiveParents} for:item="v">
+                        <c-quantic-category-facet-parent onselectvalue={onSelect} key={v.value} item={v}></c-quantic-category-facet-parent>
+                    </template>
+                    <template if:true={activeParent}>
+                        <li class="slds-var-m-horizontal_medium slds-grid">
+                            <span class="nowrap facet-text__bold slds-var-m-horizontal_small">{activeParent.value} ({activeParent.numberOfResults})</span>
```

Comment:
I would tend to renamed the CSS class from `facet-text__bold` to `facet__parent-text`. It would better match the element purpose.